external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -136,9 +151,9 @@ def _init_recipe(self):
         train_type = self._hyperparams.algo_backend.train_type
         logger.info(f'train type = {train_type}')
 
-        recipe = os.path.join(recipe_root, 'semisl.yaml')
+        recipe = os.path.join(recipe_root, 'class_incr.yaml')
         if train_type == TrainType.SemiSupervised:
-            recipe = os.path.join(recipe_root, 'semisl.yaml')
+            recipe = os.path.join(recipe_root, 'class_incr.yaml')
```

Comment:
The recipe seems to have been specified incorrectly. I think 'SemiSupervised' should raise NotImplementedError like 'SelfSupervised'

Reply:
Okay, I'll do that. Thank you.